### ISSUE TITLE
[BUG] Bad GROUP BY generation query for PostgreSQL

### ISSUE THREAD

**Issue by im-kulikov** [75team/thinkjs#461]
## DESC

@welefen: Bad GROUP BY generation query for PostgreSQL (v9.5.4).

Example:
```js
yield this.model('some_model')
                .field([
                    'COUNT("some_table"."some_field_1") AS "members_count"  ',
                    '"some_table"."some_field_2"        AS "field_2"        ',
                    ....
                    '"another_table"."some_field_2"     AS "field_2"        ',
                    '"another_table"."some_field_3"     AS "family_name"    ',
                    '"another_table"."some_field_4"     AS "family_owner_id"',
                ])
                .join('"another_table" ON (...)')
                .where(...)
                .group(['"some_table"."some_field_2"', '"another_table"."some_field_3"', '"another_table"."some_field_4"'])
                .select();
```

Generate next query:

```sql
SELECT    COUNT("some_table"."some_field_1") AS "field_1",
          "some_table"."some_field_2" AS "field_2",
          "another_table"."some_field_1N"      AS "field_1N",
          "another_table"."some_field_N"   AS "field_N"
     FROM "some_table"
LEFT JOIN "another_table"
       ON (...)
    WHERE (...)
 GROUP BY "some_table".`"some_field_2"`,
          "another_table".`"some_field_1N"`,
          "another_table".`"some_field_N"`
```

it's wrong query because it wraps field names with "`"

it must be like this:

```sql
SELECT    COUNT("some_table"."some_field_1") AS "field_1",
          "some_table"."some_field_2" AS "field_2",
          "another_table"."some_field_1N"      AS "field_1N",
          "another_table"."some_field_N"   AS "field_N"
     FROM "some_table"
LEFT JOIN "another_table"
       ON (...)
    WHERE (...)
 GROUP BY "some_table"."some_field_2",
          "another_table"."some_field_1N",
          "another_table"."some_field_N"
```

I talk about this part:
```sql
 GROUP BY "some_table".`"some_field_2"`,
          "another_table".`"some_field_1N"`,
          "another_table".`"some_field_N"`
```
it should be:
```sql
 GROUP BY "some_table"."some_field_2",
          "another_table"."some_field_1N",
          "another_table"."some_field_N"
```

## ENV

Platform:

`macOS / Linux / Win`

PostgreSQL Version:

` 9.5.4`

Node.js Version:

`v6.5.0 - v6.7.0`

ThinkJS Version:

`Release 2.2.9-2.2.12`

**Comment by im-kulikov** [comment]
If I am right, then the problem in this part of the code:
https://github.com/75team/thinkjs/blob/master/src/adapter/db/_parse.js#L521-L528

**Comment by im-kulikov** [comment]
i can try to create test-case for postgresql if you explain, how to make it.
today i push to my repo simple test-case for postgresql (like mysql), but it so simply and not include all of needed tests

**Comment by im-kulikov** [comment]
fixed and tested by #463

**Comment by im-kulikov** [comment]
Now you can use:

```js
...
.group(['"some_table"."some_field_2"', '"another_table"."some_field_3"', '"another_table"."some_field_4"'])
```
or
```js
...
.group(['some_table.some_field_2', 'another_table.some_field_3', 'another_table.some_field_4'])
```

and new future:

```js
...
.group({ 'field1': 1, 'field2': -1 }) // result: GROUP BY "field1" ASC, "field2" DESC
// OR
.group({ 'field1': 'ASC', 'field2': 'DESC' }) // result: GROUP BY "field1" ASC, "field2" DESC
// OR
.group(['field1 ASC, field2 DESC']) // result: GROUP BY "field1" ASC, "field2" DESC
```

**Comment by im-kulikov** [comment]
fixed and tested by #463
